README.md

```diff
@@ -196,13 +196,62 @@ knife ec2 server create -I ami-d0f89fb9 --ssh-key your-public-key-id -f m1.mediu
 knife ec2 server create -I ami-173d747e -G windows -f m1.medium --user-data ~/your-user-data-file -x '.\a_local_user' -P 'yourpassword' --ssh-key your-public-key-id --spot-price price-in-USD
 
 # Pass --server-connect-attribute to specify the instance attribute that we will try to connect to via ssh/winrm
-# Possible values of --server-connect-attribute: private_dns_name, private_ip_address, public_dns_name, public_ip_address
+# Possible values of --server-connect-attribute: private_dns_name, private_ip_address, dns_name, public_ip_address
 # If --server-connect-attribute is not specified, knife attempts to determine if connecting to the instance's public or private IP is most appropriate based on other settings
 knife ec2 server create -I ami-173d747e -x ubuntu --server-connect-attribute public_ip_address
 ```
 
 View additional information on configuring Windows images for bootstrap in the documentation for [knife-windows](https://docs.chef.io/plugin_knife_windows.html).
 
+
+#### Bootstrap Windows instance without user-data through winrm ssl transport
+
+Users can bootstrap the Windows instance without the need to provide the user-data. `knife-ec2` has the ability to bootstrap the Windows instance through `winrm protocol` using the `ssl` transport. This require users to set `--winrm-transport` option as `ssl`, `--winrm-authentication-protocol` option as `negotiate` and `--winrm-ssl-verify-mode` option as `verify_none`. This will reduce the users overhead as they would not be required to pass any user-data to configure the winrm for bootstrapping the instance.
+
+***Note***: Users also need to pass the `security_group` with having the required ports opened like `5986` for `winrm ssl transport`.
+
+Below is the sample command to create a Windows instance and bootstrap it through `ssl` transport without passing any user-data:
+
+```
+knife ec2 server create -N chef-node-name -I your-windows-image -f flavor-of-server -x '.\a_local_user' -P 'yourpassword' --ssh-key your-public-key-id --winrm-transport ssl --winrm-authentication-protocol negotiate --winrm-ssl-verify-mode verify_none --security-group-ids your-security-groups -VV
+```
+
+#### Bootstrap Windows instance with user-data through winrm plaintext transport
+
+Users can bootstrap the Windows instance through `winrm plaintext transport` as well. But for this users need to pass the user-data along with `--winrm-transport` option as `plaintext` and `--winrm-authentication-protocol` option as `basic`.
+
+***Note***: Users also need to pass the `security_group` with having the required ports opened like `5985` for `winrm plaintext transport`.
```

Comment:
what does this mean? can you pass the necessary security group settings in the user-data? if so, let's give that example.